core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/runtimeRoutesModels.scala

```diff
@@ -81,4 +81,6 @@ object UpdateRuntimeConfigRequest {
 final case class UpdateRuntimeRequest(updatedRuntimeConfig: Option[UpdateRuntimeConfigRequest],
                                       allowStop: Boolean,
                                       updateAutopauseEnabled: Option[Boolean],
-                                      updateAutopauseThreshold: Option[FiniteDuration])
+                                      updateAutopauseThreshold: Option[FiniteDuration],
+                                      labelsToUpsert: LabelMap,
+                                      labelsToDelete: List[String])
```

Comment:
Wonder if this should be a `Set[String]`

Reply:
oh actually yeah that seems better👍